tests/unit/cancelable-promise-helpers-test.js

```diff
@@ -307,6 +307,31 @@ test("hash cancels children if parent is canceled", function(assert) {
   assert.equal(obj.get('child.concurrency'), 0);
 });
 
+test("yieldable helpers work with null/undefined values", function(assert) {
```

Comment:
This only tests the `hash` helper, but the code path is common across all helpers. I could write tests for the other helpers, but it seemed unnecessary -- let me know if you think otherwise.